Deploy/kubernetes/README.md

```diff
@@ -0,0 +1,102 @@
+
+# Integration with Kubernetes
+
+Firstly you need to get docker images of harbor.You can get it by the two ways:
```

Comment:
s/First/Firstly

Note, you need to add a space between  '.' and latter; for example, `of harbor. You can`